pachyderm/templates/etcd/storageclass.yaml

```diff
@@ -0,0 +1,21 @@
+{{- if and (.Values.etcd.storageClass | not) (gt .Values.etcd.dynamicNodes 0.0) (eq .Values.pachd.storage.backend "LOCAL" | not) -}}
```

Comment:
<note for kyle> check is current manifest has storage class for local (I'm assuming not)
We can probably take out the etcd dymanic nodes check since we're failing on the value above

Reply:
The local current local manifest does not have the `etcd` StorageClass.

Refactored the check, too.